main.go

```diff
@@ -55,9 +56,11 @@ func main() {
 	// If not enabled multithreading, server will run on a single core.
 	var numCores int
 	if config.EnableMultiThreading {
```

Comment:
Can this be renamed to IsMultithreadingEnabled ? This will make is more readable.

Reply:
This wasn't added as part of this PR. Lets address this in separate PR.